integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/CarbonRDD.scala

```diff
@@ -59,6 +76,33 @@ abstract class CarbonRDD[T: ClassTag](@transient sc: SparkContext,
       map(f => CarbonProperties.getInstance().addProperty(f._1, f._2))
     internalCompute(split, context)
   }
+
+  private def getConf: Configuration = {
+    val configuration = new Configuration(false)
+    val bai = new ByteArrayInputStream(CompressorFactory.getInstance().getCompressor
+      .unCompressByte(confBytes))
+    val ois = new ObjectInputStream(bai)
+    configuration.readFields(ois)
+    ois.close()
+    configuration
+  }
+
+  private def setS3Configurations(hadoopConf: Configuration): Unit = {
```

Comment:
Can you use the same function in CarbonInputFormatUtil. It was added when I rebase yesterday